flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlToOperationConverterTest.java

```diff
@@ -1660,6 +1665,65 @@ public void testReset() {
         assertThat(((ResetOperation) operation2).getKey()).hasValue("test-key");
     }
 
+    @Test
+    public void testHelpCommands() {
+        ExtendedParser extendedParser = new ExtendedParser();
+        Stream.of("HELP", "HELP;", "HELP ;", "HELP\t;", "HELP\n;")
+                .forEach(
+                        command -> {
+                            Operation operation1 =
+                                    extendedParser
+                                            .parse(command)
+                                            .orElseThrow(
+                                                    () ->
+                                                            new RuntimeException(
+                                                                    "Fail to parse '"
+                                                                            + command
+                                                                            + "'"));
+                            assertThat(operation1).isInstanceOf(HelpOperation.class);
+                        });
+    }
+
+    @Test
+    public void testClearCommands() {
+        ExtendedParser extendedParser = new ExtendedParser();
+        Stream.of("CLEAR", "CLEAR;", "CLEAR ;", "CLEAR\t;", "CLEAR\n;")
+                .forEach(
+                        command -> {
+                            Operation operation1 =
+                                    extendedParser
+                                            .parse(command)
+                                            .orElseThrow(
+                                                    () ->
+                                                            new RuntimeException(
+                                                                    "Fail to parse '"
+                                                                            + command
+                                                                            + "'"));
+                            assertThat(operation1).isInstanceOf(ClearOperation.class);
+                        });
+    }
+
+    @Test
+    public void testQuitCommands() {
+        ExtendedParser extendedParser = new ExtendedParser();
+        Stream.of(
+                        "QUIT", "QUIT;", "QUIT ;", "QUIT\t;", "QUIT\n;", "EXIT", "EXIT;", "EXIT ;",
+                        "EXIT\t;", "EXIT\n;")
+                .forEach(
+                        command -> {
+                            Operation operation1 =
+                                    extendedParser
+                                            .parse(command)
+                                            .orElseThrow(
```

Comment:
You can remove the `orElseThrow` with a proper assertj assertion:

```
assertThat(extendedParser.parse(command))
  .get() // Returns an assertion on the value, and throws proper AssertionException if no value is present
  .isInstanceOf(QuitOperation.class);
```

Reply:
thanks, done